src/index.ts

```diff
@@ -37,13 +41,21 @@ expressApp.listen(7000, () => {
   logger.info(`Start readiness check at :${PORT}/status`);
 });
 
+
 /**
  * The main Kafka subscription
  */
 if (ENABLED) {
   (async () => {
     await connectMongo();
 
+    const rollCallClient = applyRollcallClient({
```

Comment:
lol should this just be `createRollcallClient`? I named my other thing `apply` because I was passing it a applies an endpoint to an existing general purpose express app

Reply:
lol was a copy pasta, will change :+1: